pages/resources.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import Head from "pages/head";
+import { Box, Container, Link } from "@material-ui/core";
+
+function Resources() {
+  return (
+    <div>
+      <Head />
+      <Container>
+        <Box maxWidth={680} mx="auto">
+          <h2>Resources</h2>
```

Comment:
Recommend bumping this to an `h1` tag, and the `h3` tags below to `h2` to keep the order consistent.

Reply:
Thank you! Making the change.